code/modules/projectiles/projectile.dm

```diff
@@ -908,7 +908,10 @@ So if we are on the 32th absolute pixel coordinate we are on tile 1, but if we a
 			feedback_flags |= (BULLET_FEEDBACK_FIRE)
 
 	if(proj.ammo.flags_ammo_behavior & AMMO_SUNDERING)
-		adjust_sunder(proj.sundering)
+		if(proj.ammo.flags_ammo_behavior & AMMO_HARD_SUNDERING)
```

Comment:
I would put this bit of code in adjust sunder itself, imo. Sunder is only projectiles right now, but might not always be. You can make hard sunder an arg.

Reply:
Explosions sunder is applied in explosions code, projectile sunder is applied in projectile code, idk why would you want to split it

Reply:
?
They both use the same proc do they not? I'm saying put the code in that proc so it's not split like you currently have it.

Reply:
they both use apply_sunder proc, im just changing the value applied. Why would i want to bring projectile code into the xenoproc?

Reply:
It doesn't matter, someone else can do it in the future, but I meant adding an arg to apply_sunder on whether to respect the sunder limit or not. Then in projectiles, the arg is true/false based on the flag. But it would mean other sunder sources could also use it in the future more readily.